docs/security-considerations.rst

```diff
@@ -216,6 +216,23 @@ Minor Details
   look like the number ``1`` as far as ``x`` is concerned, but ``msg.data`` will
   be different, so if you use ``keccak256(msg.data)`` for anything, you will get different results.
 
+
+*******************
+Formal Verification
+*******************
+
+Using formal verification, it is possible to perform an automated mathematical
+proof that your source code fulfills a certain formal specification.
+The specification is still formal (just as the source code), but usually much
+simpler. There is a prototype in Solidity that performs formal verification and
```

Comment:
The *There is a prototype in Solidity* part isn't true anymore. Why3 was never finished and was removed a release ago.